### ISSUE TITLE
CREATE STREAM fails when field name is "group"

### ISSUE THREAD

**Issue by ftrossbach** [confluentinc/ksql#273]
Hi,

I tried to let KSQL loose on a topic that contains  ([RSVP data from Meetup.com](https://www.meetup.com/meetup_api/docs/stream/2/rsvps/#websockets)) as data in JSON format. 

One of the fields in this dataset is called "group", which is also a keyword in KSQL (GROUP by). The parser fails with 

```
ksql> CREATE STREAM rsvps2 (group VARCHAR) WITH (value_format='json', kafka_topic='rsvps');
line 1:23: extraneous input 'group' expecting {'ADD', 'APPROXIMATE', 'AT', 'CONFIDENCE', 'NO', 'SUBSTRING', 'POSITION', 'TINYINT', 'SMALLINT', 'INTEGER', 'DATE', 'TIME', 'TIMESTAMP', 'INTERVAL', 'YEAR', 'MONTH', 'DAY', 'HOUR', 'MINUTE', 'SECOND', 'ZONE', 'OVER', 'PARTITION', 'RANGE', 'ROWS', 'PRECEDING', 'FOLLOWING', 'CURRENT', 'ROW', 'VIEW', 'REPLACE', 'GRANT', 'REVOKE', 'PRIVILEGES', 'PUBLIC', 'OPTION', 'EXPLAIN', 'ANALYZE', 'FORMAT', 'TYPE', 'TEXT', 'GRAPHVIZ', 'LOGICAL', 'DISTRIBUTED', 'TRY', 'SHOW', 'TABLES', 'SCHEMAS', 'CATALOGS', 'COLUMNS', 'COLUMN', 'USE', 'PARTITIONS', 'FUNCTIONS', 'TO', 'SYSTEM', 'BERNOULLI', 'POISSONIZED', 'TABLESAMPLE', 'RESCALED', 'ARRAY', 'MAP', 'SET', 'RESET', 'SESSION', 'DATA', 'START', 'TRANSACTION', 'COMMIT', 'ROLLBACK', 'WORK', 'ISOLATION', 'LEVEL', 'SERIALIZABLE', 'REPEATABLE', 'COMMITTED', 'UNCOMMITTED', 'READ', 'WRITE', 'ONLY', 'CALL', 'NFD', 'NFC', 'NFKD', 'NFKC', 'IF', 'NULLIF', 'COALESCE', IDENTIFIER, DIGIT_IDENTIFIER, QUOTED_IDENTIFIER, BACKQUOTED_IDENTIFIER}
```
I tried to escape the name by using double quotes, but then KSQL seems to include those quotes in the name and never finds any data.

**Comment by ftrossbach** [comment]
Looking at the ANTLR grammar, quotes seem to be supported. Is it an issue that I try to assign sub json object to a string? The group element in Meetup's data is a json object with different field types. Somehow I assumed I just could use this and then use the JSONPath function to extract further data. I'm probably doing it wrong.

**Comment by hjafarpour** [comment]
@ftrossbach Thanks for trying KSQL and your feedback.
As you mentioned GROUP is a keyword in the grammar and cannot be used as column name. Although the quoted identifier is defined in the grammar we do not support it completely yet. 
There are couple of ways you can use your JSON. Of course the easy way would be to change the column name to some thing like '_group'. If it is not possible you can use our EXTRACTJSONFIELD to extract the fields you need from JSON and create a new stream with acceptable column names.
Let me know if this works for you.

**Comment by rmoff** [comment]
Tracking in #677
